lib/new_relic/language_support.rb

```diff
@@ -83,6 +83,10 @@ def camelize_with_first_letter_downcased(string)
       camelized[0].downcase.concat(camelized[1..-1])
     end
 
+    def snakeize(string)
+      string.gsub(/(.)([A-Z])/, '\1_\2').downcase
+    end
```

Comment:
Nice! We have a language support test file. Could you add tests for this method to it?

Reply:
added a test for this here https://github.com/newrelic/newrelic-ruby-agent/pull/2203/commits/436f5d26cabaa70775d009a80d8be99d85ad16ea